src/components/Ublock/Ublock.tsx

```diff
@@ -43,7 +45,7 @@ const Ublock: React.FC<UblockProps> = ({
     return (
         <div
             id={id}
-            className={`${blockClassName} ${colorClassname}  ${wavesClassname} ${width} ${largeClassname} ${alertClassname} ${wideClassName} ${noPaddingClassname}`}
+            className={`${blockClassName} ${className} ${colorClassname}  ${wavesClassname} ${width} ${largeClassname} ${alertClassname} ${wideClassName} ${noPaddingClassname}`}
```

Comment:
className could be undefined, it's better to create a variable based on it first, like:
```suggestion
          const customClassName = className ? className : ''
```
and then use the variable .